R/pr.R

```diff
@@ -967,22 +969,22 @@ pr_branch_delete <- function(pr) {
 
   if (is.null(pr_ref)) {
     ui_bullets(c(
-      "i" = "PR {.val {pr$pr_string}} originated from branch {.val {pr_remref}},
+      "i" = "{.href [PR {pr$pr_string}]({pr$pr_html_url})} originated from branch {.val {pr_remref}},
```

Comment:
```suggestion
      "i" = "{.href [PR {pr$pr_number}]({pr$pr_html_url})} originated from branch {.val {pr_remref}},
```

This looks a bit bad.
![image](https://github.com/r-lib/usethis/assets/52606734/7ad713b7-3c86-4c61-9901-88f3be306f20)

Any reason on why not just including the PR Number?